test/Twig/Tests/EnvironmentTest.php

```diff
@@ -498,33 +498,6 @@ public function testAddRuntimeLoader()
         $this->assertEquals('foo', $twig->render('func_string_named_args'));
     }
 
-    /**
-     * @expectedException \Twig\Error\RuntimeError
-     * @expectedExceptionMessage Circular reference detected for Twig template "base.html.twig", path: base.html.twig -> base.html.twig in "base.html.twig" at line 1
```

Comment:
so, no such protection against circular references anymore ?

Reply:
No,  as we don't load templates recursively, so not needed anymore. And when the parent is dynamic, we didn't have such protection anyway (same for includes).